graylog2-web-interface/src/components/lookup-tables/Cache.tsx

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (C) 2020 Graylog, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the Server Side Public License, version 1,
+ * as published by MongoDB, Inc.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * Server Side Public License for more details.
+ *
+ * You should have received a copy of the Server Side Public License
+ * along with this program. If not, see
+ * <http://www.mongodb.com/licensing/server-side-public-license>.
+ */
+import * as React from 'react';
+import { useHistory } from 'react-router-dom';
+import { PluginStore } from 'graylog-web-plugin/plugin';
+
+import Routes from 'routing/Routes';
+import { Row, Col, Button } from 'components/bootstrap';
+import useScopePermissions from 'hooks/useScopePermissions';
+import type { LookupTableCache } from 'logic/lookup-tables/types';
+
+import {
+  SummaryContainer,
+  SummaryRow,
+  Title,
+  Value,
+} from './caches/SummaryComponents.styled';
+
+type Props = {
+  cache: LookupTableCache,
+};
+
+const Cache = ({ cache }: Props) => {
+  const history = useHistory();
+  const { loadingScopePermissions, scopePermissions } = useScopePermissions(cache);
+  const plugins = {};
+
+  PluginStore.exports('lookupTableCaches').forEach((p: any) => {
+    plugins[p.type] = p;
+  });
+
+  const plugin = plugins[cache.config.type];
+
+  if (!plugin) {
+    return <p>Unknown cache type {cache.config.type}. Is the plugin missing?</p>;
+  }
+
+  const summary = plugin.summaryComponent;
+
+  const handleEdit = (cacheName: string) => () => {
+    history.push(Routes.SYSTEM.LOOKUPTABLES.CACHES.edit(cacheName));
+  };
+
+  return (
+    <Row className="content">
+      <Col md={12}>
+        <h2>{cache.title} <small>({plugin.displayName})</small></h2>
+        <SummaryContainer>
+          <SummaryRow>
+            <Title>Description:</Title>
+            <Value>{cache.description || <em>No description.</em>}</Value>
```

Comment:
In cases like this one we usually use the nullish coalescing operator. In this specific case it will probably never make a difference, but it is a good practice to make sure we would e.g. render the description when it is `0`.

Reply:
What if the description is an empty string?

Reply:
Good point, I think ideally the backend should not return an empty string, but in this case I would also just use `||`.